KirbySamurai.re

```diff
@@ -52,21 +52,20 @@ let createElement = (~children as _, ()) =>
     (hooks,
       switch(state.gameState) {
       | Stop => (
-        <View 
-        ref={r => Focus.focus(r)}
```

Comment:
The `KeyboardInput` at least lets us remove this ref/focus handling (handles it under the hood for us). Since the `<KeyboardInput />` also grabs input if its lost - it seemed to work more reliably for me when clicking on the screen.

Reply:
Ah!I like that `<Keyboard />` handles ref/focus under the hood!